app/models/task.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :uuid             not null, primary key
+#  name        :string
+#  description :string
+#  state       :integer          default(0)
+#  project_id  :uuid
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_tasks_on_project_id  (project_id)
+#
+
+class Task < ActiveRecord::Base
+  belongs_to :project
+
+  enum state: [:todo, :in_progress, :done]
```

Comment:
I see what you guys are saying about implicit data in the ordering. If it were a more complex model then I would have considered the hash notation.